ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/resources/SearchResult.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.api.resources;
+
+import org.eclipse.che.api.project.shared.Constants;
+import org.eclipse.che.api.project.shared.SearchOccurrence;
+import org.eclipse.che.api.project.shared.dto.SearchOccurrenceDto;
+import org.eclipse.che.api.project.shared.dto.SearchResultDto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Vitalii Parfonov
```

Comment:
Actually class represents a POJO with a readable name.
javadoc is not necessary